### PR TITLE
added SSP playbook

### PR DESCRIPTION
**Pull request by Knickkennedy**
attempt deuce with the sign-off added

### CODE REVIEW

**Comment by Knickkennedy** [comment]
Okay let's try this again!

**Comment by hollisc** [comment]
Please resolve existing conflict and this will have to wait until the other PRs are merged.

File: 0-bootstrap/single-cluster/1-infra/argocd/namespace-sterling-sp.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    gitops.tier.layer: infra
+  annotations:
+    argocd.argoproj.io/sync-wave: "100"
+spec:
+  destination:
+    namespace: openshift-gitops
+    server: https://kubernetes.default.svc
+  project: infra
+  source:
+    path: namespaces/sterling-sp
```
Comment (hollisc) [inline 0-bootstrap/single-cluster/1-infra/argocd/namespace-sterling-sp.yaml]: The path should be `namespaces/ssp`

File: 0-bootstrap/single-cluster/1-infra/kustomization.yaml [hunk 2]
```diff
@@ -26,6 +26,7 @@ resources:
 #- argocd/namespace-b2bi-prod.yaml
 #- argocd/namespace-b2bi-nonprod.yaml
 #- argocd/namespace-pem.yaml
+#- argocd/namespace-sterling-sp.yaml
```
Comment (hollisc) [inline 0-bootstrap/single-cluster/1-infra/kustomization.yaml]: Adjust the name to match the namespace so it should be `namesapce-ssp.yaml`

**Review by Knickkennedy — COMMENTED** [review]
I think this fixes everything @hollisc

**Review by hollisc — APPROVED** [review]
Looks good

**Pull request merged**